### DISCUSSION TITLE
Question: Why is WinUI 3's scroll experience is far from WinUI 2

### DISCUSSION THREAD

**Discussion by dylech30th** [microsoft/microsoft-ui-xaml#7444] [Q&A]
Open both [WinUI 3 Control Gallery](https://www.microsoft.com/en-us/p/winui-3-controls-gallery/9p3jfpwwdzrc) and [WinUI 2 Control Gallery](https://www.microsoft.com/en-us/p/xaml-controls-gallery/9msvh128x2zt)'s Settings page, it seems that the WinUI3 version only got 30fps while the WinUI 2 version has an excellent scroll experience with an adaptive framerate that correlated to the screen's refresh rate, this also happens in my own app built with WinUI 3. If you are using an high-refresh-rate screen, the difference would be even more obvious

**Comment by LasmGratel** [comment]
same issues here while on Windows 10 21H1 and Windows 11

**Comment by kmgallahan** [comment]
Dupe #2513

**Comment by dylech30th** [comment]
Thank you for the reply, I've just confirmed that the scroll experience in Windows App SDK 1.0 is good *only if you are using the mouse wheel*, it seems that the framerate is still locked to 30fps when you are dragging the slider of the scroll bar instead of using the mouse wheel, and that is not gonna happen in WinUI 2

**Comment by dylech30th** [comment]
You can find the videos here and the difference is so obvious
ListView Scrolling in UWP(WinUI 2.6): [Video link](https://sora.ink/wp-content/uploads/2021/09/Desktop-2021.09.10-18.14.09.04.mp4)
in contrast to
ListView Scrolling in WinUI 3 Desktop(Windows App SDK 1.0.0 Experimental): [Video link](https://sora.ink/wp-content/uploads/2021/09/Desktop-2021.09.10-18.34.15.05.mp4)

**Comment by sylveon** [comment]
https://github.com/microsoft/microsoft-ui-xaml/issues/4690

**Comment by dylech30th** [comment]
I'm aware of that, thanks for the reply

**Comment by StephenLPeters** [comment]
@Austin-Lamb @bartekk8 and @RBrid FYI

**Comment by marb2000** [comment]
I'm just curious if this is still an issue. I'm testing the latest WinUI 3 and 2 Gallery and they seem very similar to me. But perhaps I'm wrong. @dylech30th, @sylveon any updated feedback?

**Comment by sylveon** [comment]
I just tried both side by side and while it doesn't appear that the WinUI 3 gallery is capped to 60fps anymore, the WinUI 2 gallery still feels smoother to scroll through.

**Comment by lhak** [comment]
I can confirm that scrolling in the "All Controls" section of the winui gallery is extremly sluggish compared to the winui 2 version.

**Comment by cultab** [comment]
Related to #6705?

Necroing this thread cause this is still an issue and is _very_ annoying
